mkdocs-base.yml

```diff
@@ -147,7 +147,7 @@ nav:
   - Welcome to Percona Everest: index.md 
   - Release Notes:
       - Release notes index: release-notes/release_notes_index.md
-      - Percona-Everest 1.4.0 (2024-12-18): release-notes/Percona-Everest-1.4.0-(2024-12-18).md
+      - Percona-Everest 1.4.0 (2024-12-15): release-notes/Percona-Everest-1.4.0-(2024-12-15).md
```

Comment:
As usual, commenting so that we don't forget to update the date.

Reply:
Sure @recharte